src/inject/generic/core/lyrics.js

```diff
@@ -52,9 +54,28 @@ window.addEventListener('load', () => {
     animate = false;
   };
 
+  const scrollHandler = (state) => {
```

Comment:
This isn't so much a `scrollHandler` as a `scrollSettingsHandler`, they are semantically different